resources/js/components/utils/formatting.ts

```diff
@@ -0,0 +1,8 @@
+/**
+ * Formats the date as "YYYY-MM-DD hh:mm:ss".
+ * https://xkcd.com/1179/
+ */
+export function formatDateTime(t: Date) {
```

Comment:
ja bym jednak poprosił datę po jęzuku aplikacji przez intl

https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Intl/DateTimeFormat

Reply:
a no i jak coś, BE udostępnia wyszystkie daty jako UTC, FE musi uwzględniać strefę czasową

Reply:
No niestety nie Intl, bo polski format jest coś w rodzaju d.mm.yyyy, czyli wypisany w kolumnie wygląda po prostu obleśnie. W nagłówku strony to jeszcze ewentualnie mógłby być, ale w kolumnie to bardzo zły pomysł. Więc dla spójności lepiej użyć *dobrego* formatu zamiast polskiego. Byłby potrzebny jeszcze jakiś format daty z dniem tygodnia, i myślę że naprawdę możemy się zdecydować na jakiś *dobry* format a nie polegać co tam komuś się wydawało że w Polsce się robi.

Co do strefy czasowej to faktycznie zrąbałem, znaczy z backendu przychodzi string UTC i parsuje się poprawnie, ale toISOString zwraca z powrotem czas UTC.

Reply:
format d.mm.yyyy podobnie jak liczby można wyrównywać do prawej, ale jak już chcemy dd.mm.yyyy
```js
const dateTimeFormat1 = new Intl.DateTimeFormat('pl-pl',
{weekday: 'long', year: 'numeric', month: 'numeric', day: '2-digit'});
dateTimeFormat1.format(new Date(2012, 5));
// piątek, 01.06.2012
```
Więc generalnie tak intl, może nie zawsze z domyślnymi parametrami

i do formatowania liczb z separatorem tysięcy też intl

Reply:
Ania by wolała daty z myślnikiem :D

no temat jeszcze do ustalenia w takim razie, ale ja, to zakładałem, że zawsze będzie intl

bo i tak w excelu pracują na takim formacie, meh

Reply:
No to na razie jest Intl z dwucyfrowym dniem. I naprawił się błąd ze strefą czasową.